client/components/RestaurantSearchResult.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+// import Result from '../containers/Result';
+import LocationContainer from '../containers/LocationContainer';
+// import Icon from '../components/Icon';
```

Comment:
pop out this Icon

Reply:
Sorry to miss these—thanks for catching them.